_pages/dev/state_management/state-persistence.md

```diff
@@ -0,0 +1,143 @@
+---
+title: State persistence
+---
+
+In version 2.0 we introduced new mechanism to persist state of the application. Previously we only had very simple and declarative mechanism called storageSync where you could provide store properties keys to persist and those persisted keys were used on application start to set initial state in the store.
+
+## API
+
+Heart of the new feature is the `StatePersistenceService` and specifically its `syncWithStorage` method. First let's go through all the options that you can pass to this function:
+
+- `key` is used to distinguish one feature state persistence from another one. Example: to store active cart id I would use `cart` key and for user session data `session`.
```

Comment:
It sounds opinionated. Can we emphasize the name of the key doesn't have to be in line with the key of the ngrx branch state? 
Maybe I'm missing something and it's the opposite so they HAVE to be the same?

Reply:
I updated it a bit to not suggest a relation with ngrx. It doesn't have to be the same.